xmtp_cryptography/src/basic_credential.rs

```diff
@@ -0,0 +1,336 @@
+use std::io::BufReader;
+
+use ed25519_dalek::{Digest as _, Signature, SigningKey, VerifyingKey};
+use k256::schnorr::CryptoRngCore;
+use openmls_basic_credential::SignatureKeyPair;
+use openmls_traits::signatures::Signer;
+use openmls_traits::{signatures, types::SignatureScheme};
+use serde::de::Error;
+use sha2::Sha512;
+use tls_codec::SecretTlsVecU8;
+use zeroize::Zeroizing;
+
+/// DO NOT CHANGE. SIGNATURES WILL BREAK
+const INSTALLATION_KEY_SIGNATURE_CONTEXT: &[u8] = b"IDENTITY UPDATE SIGNATURE";
```

Comment:
Feels like this should get passed in, given it’s in a whole other crate from where this context comes from.